system/jlib/jptree.cpp

```diff
@@ -6863,9 +6868,8 @@ class CPullJSONReader : public CJSONReaderBase<X>, implements IPullPTreeReader
             expecting(",");
         return true;
     }
-    void newAttribute()
+    void newNamedAttribute()
     {
-        readName(tag.clear());
         skipWS();
```

Comment:
It doesn't look like skipWS is ever necessary on a call to newNamedAttribute() ... ?